src/logging_unittest.cc

```diff
@@ -449,6 +451,22 @@ void TestLoggingLevels() {
   LogWithLevels(1, GLOG_FATAL, false, true);
 }
 
+int TestVlogHelper() {
+  if (VLOG_IS_ON(1)) {
+    return 1;
+  }
+  return 0;
+}
+
+void TestVLogModule() {
+  int c = TestVlogHelper();
+  EXPECT_EQ(0, c);
+
+  EXPECT_EQ(0, SetVLOGLevel("logging_unittest", 1));
+  c = TestVlogHelper();
+  EXPECT_EQ(1, c);
```

Comment:
The test fails on this line.

Reply:
Likely related to this:

https://github.com/google/glog/blob/0b3d4cb471e36d8e677e75e4e8d0345e7bf2a418/src/glog/vlog_is_on.h.in#L90-L92

Reply:
I removed the test from non __GNUC__ environments but it still fails for mingw.

Reply:
I fixed the tests for mingw environment as well